horovod/common/message/mpi_message.cc

```diff
@@ -15,67 +15,14 @@
 // =============================================================================
 
 #include "mpi_message.h"
-#include "wire/mpi_message_generated.h"
+
 #include <iostream>
 
+#include "../wire/mpi_message_generated.h"
+
```

Comment:
Do we need to have mpi_message.cc and message.cc?  Why not just move everything to message.cc?  If we add Gloo backend or for coordination, we can still serialize those generic messages.